app/controllers/mixins/checked_id_mixin.rb

```diff
@@ -10,7 +10,22 @@ def checked_item_id(hash = params)
     # Common routine to find checked items on a page (checkbox ids are
     # "check_xxx" where xxx is the item id or index)
     def find_checked_items(prefix = nil)
-      if params[:miq_grid_checks].present?
+      has_no_check = true
```

Comment:
This line can be dropped now as line 23 below will set true or false accordingly.